tasks.go

```diff
@@ -22,17 +24,21 @@ func GetAllTasks(tarFile *Archive) ([]Task, error) {
 		return nil, err
 	}
 
+	if len(projects) == 0 {
+		return []Task{}, errors.New("no projects visible to the currently logged in user")
```

Comment:
@rhcarvalho opened a PR to address that, here: https://github.com/feedhenry/fh-system-dump-tool/pull/16